### PR TITLE
Add Edge API references for getLocationHint and setLocationHint

### PR DESCRIPTION
**Pull request by kevinlind**
## Description

Add API references for Edge.getLocationHint and Edge.setLocationHint.
Add release notes for iOS Edge v1.5.0 and Android Edge v1.4.0

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/edge-network-extensions/api-reference.md [hunk 1]
```diff
@@ -50,6 +50,37 @@ Resets current state of the AEP Edge extension and clears previously cached cont
...
+iOS (AEP 3.x)
+
+<Tabs query="platform=ios-aep&api=send-event"/>
+
+## setLocationHint
+
+Sets the Edge Network location hint used in requests to the Adobe Experience Platform Edge Network. Passing nil or an empty string clears the existing location hint. Edge Network responses may overwrite the location hint to a new value when necessary to manage network traffic.
+
+<InlineAlert variant="warning" slots="text"/>
+Use caution when setting the location hint. Only use location hints for the "EdgeNetwork" scope. An incorrect location hint value will cause all Edge Network requests to fail with 404 response code.
```
Comment (calebk1) [inline src/pages/documentation/edge-network-extensions/api-reference.md:72]: Just a small thing, but I think the renderer is a bit finnicky and it needs the newline between the <InlineAlert> and the text!

**Review by calebk1 — COMMENTED** [review]
Looks good to me!

I think we just need some newlines between some of the blocks for the renderer? But if you've tested this locally and it doesn't then we're good to go.

File: src/pages/documentation/edge-network-extensions/tabs/api-reference.md [hunk 2]
```diff
@@ -147,6 +147,72 @@ NSDictionary *data = @{ @"sample" : @"data"};
 }];
 ```
 
+<Variant platform="android" api="get-location-hint" repeat="6"/>
```
Comment (emdobrin) [inline src/pages/documentation/edge-network-extensions/tabs/api-reference.md:150]: nit - can we keep tabs in alphabetical order too?

**Review by emdobrin — APPROVED** [review]
Looks great, just a small comment

File: src/pages/documentation/edge-network-extensions/api-reference.md (hunk 1 above)
Comment (kevinlind) [reply src/pages/documentation/edge-network-extensions/api-reference.md:73]: I tested locally and it looked fine. I added the newline, though. If nothing else it makes the raw file easier to read with the newline.

**Pull request merged**